05-documentation.Rmd

```diff
@@ -0,0 +1,110 @@
+# Code Documentation
+
+## Introduction
+
+Documentation of code is essential, even if the only person using your code 
+is future-you. Use 
+[roxygen2](https://github.com/klutometis/roxygen) with enabled [markdown](https://github.com/klutometis/roxygen/blob/master/vignettes/markdown.Rmd)
+support to keep the documentation close to your code.
+
+```{r, include=FALSE}
+knitr::opts_chunk$set(eval = FALSE)
+```
+
+## Title and Description
+For the title, describe concisely what the function does at the very 
+top of your function documentation. Do not use a dot at the end of the title 
+and use title style capitalization. For both the title and the description, 
+there is no need to use the explicit `@title` or `@description` tag. To 
+visually separate the title and the description, leave one line blank after 
+each of them.
+```{r}
+#' Combine Values into a Vector or List
+#' 
+#' This is a generic function which combines its arguments.
+#'
+```
+
```

Comment:
I'd add something like:

The only time you should need to use an explicit `@description` is when the description is multiple paragraphs or includes more complex formatting like a bulleted list.

(And then add an example)